app/views/orga/teams/show.html.slim

```diff
@@ -82,6 +84,17 @@ nav.btn-group
       = links_to_conferences([preference.conference]).first
       = preference_option(preference)
 
+- if can? :see_offered_conferences, @team
+
+  - if @team.conference_attendances.present?
+    h4 Conference offers
+    dl
+      - @team.conference_attendances.each do |offer|
+        dt Offered conference:
+        dd = offer&.conference&.name
```

Comment:
I'm a bit torn about the lonely operator here 🤔  You already check whether or not there's a `ConferenceAttendance` record and afaik Rails 5 automatically enforces `belongs_to` associations, so there will always be a conference associated plus the conference has a validation on the presence of its name... For the sake of simplicity I would leave the lonely operators here.